### ISSUE TITLE
Make class docstrings available in gui?

### ISSUE THREAD

**Issue by ceball** [ioam/paramnb#64]
Nowhere in paramnb is the class docstring displayed. In a notebook you can always do e.g. ``Class??`` to see Class's docstring, but it might be nice (particularly when choosing objects in a dropdown) to be able to see the docstring in the gui.

**Comment by jbednar** [comment]
I agree that would be helpful.  We have to be careful, because class docstrings can get very long, and if we display them when hovering it's possible to make the interface unusable by covering up important info.  As long as we do it well, it would be great to have this!
